src/main/java/com/flightstats/hub/cluster/ClusterCacheDao.java

```diff
@@ -79,27 +78,20 @@ public ContentPath get(String name, ContentPath defaultPath, String basePath) {
         }
     }
 
-    private ContentPath get(String path) throws Exception {
-        byte[] bytes = curator.getData().forPath(path);
-        String found = new String(bytes, StandardCharsets.UTF_8);
-        trace(path, "get found {}", found);
-        return ContentPath.fromUrl(found).get();
-    }
-
-    public void updateDecrease(ContentPath nextPath, String name, String basePath) {
-        update(nextPath, name, basePath, (existing) -> nextPath.compareTo(existing.key) < 0);
+    public void setIfBefore(ContentPath nextPath, String name, String basePath) {
+        setPathValueIf(nextPath, name, basePath, (existing) -> nextPath.compareTo(existing.key) < 0);
     }
 
-    public void updateIncrease(ContentPath nextPath, String name, String basePath) {
-        update(nextPath, name, basePath, (existing) -> nextPath.compareTo(existing.key) > 0);
+    public void setIfAfter(ContentPath nextPath, String name, String basePath) {
```

Comment:
I can't figure out what after means. After what? I don't see an integer or timestamp, so I'm not sure how to interpret 'after'. Maybe setIfNewer() or setIfMoreRecent() or some other name might be more descriptive?

If this is just me being ignorant of a term common to the hub, then ignore me.

Reply:
"Newer"/"older" would work as well.  There's sort of a general expectation in hub code that "paths" have an intrinsic time so "after" and "newer" are synonymous to me within that context.

Reply:
My 2c is that "newer" is a little bit clearer in that it kind of connotes the idea of comparing two items. With the hub being so time-centric, it's a little confusing about which after we're talking about (stable time, order of items, ttl...) if I don't stop to think about it.

Reply:
Sounds good.  "Newer"/"older" it is then.